{"name": "test_003_panic_and_power_loss", "status": "broken", "statusDetails": {"message": "TypeError: comment() takes 1 positional argument but 2 were given", "trace": "@pytest.fixture\n    def pre_conditions():\n        \"\"\"Preconditions\"\"\"\n        Report.comment('***************Pre Conditions***************')\n        # - Turn on ignition, and wait for 30-60 seconds\n        Report.comment('Ignition turned On.')\n        Report.comment('Wait time of 50 seconds.')\n        sleep(0.5)\n        yield\n>       Report.comment('******************Tear Down*****************', True)\nE       TypeError: comment() takes 1 positional argument but 2 were given\n\nXirgo_KB1_Scripts\\Panic\\test_003_panic_and_power_loss.py:17: TypeError"}, "attachments": [{"name": "stdout", "source": "3a2f93f0-60fe-431a-8af9-c6ecace4121d-attachment.txt", "type": "text/plain"}], "start": 1637888375025, "stop": 1637888377040, "uuid": "6ca2db46-25af-4aec-b3fe-3a0744b7d0c1", "historyId": "9b73afe4f6d784d4d0e04292f62f2a97", "testCaseId": "ad0db2d4bed47ab470477f94a5860153", "fullName": "Xirgo_KB1_Scripts.Panic.test_003_panic_and_power_loss#test_003_panic_and_power_loss", "labels": [{"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "Xirgo_KB1_Scripts.Panic"}, {"name": "suite", "value": "test_003_panic_and_power_loss"}, {"name": "host", "value": "WX-AMADRID"}, {"name": "thread", "value": "19376-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Xirgo_KB1_Scripts.Panic.test_003_panic_and_power_loss"}]}